scripts/helm/verify-kind-images.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+
+SCRIPT_DIR="$(dirname "$(realpath "${BASH_SOURCE[0]:-"$0"}")")"
+ROOT_DIR="$SCRIPT_DIR/../.."
+CHART_DIR="$ROOT_DIR/chart"
+HELM_TPL_IMAGES="$CHART_DIR/images.txt"
+HELM_DEP_IMAGES="$CHART_DIR/dependencies-images.txt"
+TMP_KIND="/tmp/kind/mayastor"
+KIND_IMAGES="kind-images.txt"
+DOCKER="docker"
+
+echo_stderr() {
+  echo -e "${1}" >&2
+}
+
+die() {
+  local _return="${2:-1}"
+  echo_stderr "$1"
+  exit "${_return}"
+}
+
+grep_image() {
+  local image="$1"
+  local file="$2"
+
+  grep -q "^$image$" "$file" || grep -q "^${image##docker.io/}$" "$file"
```

Comment:
The Hostpath LocalPV images don't start with "docker.io".

Reply:
Ok, are you suggesting any changes here?

Reply:
Right, sometimes the registry might not be specified, do we handle those cases?

Reply:
I'm confused.. it's handled in line 29, what's the problem?

Reply:
I might have misinterpreted the line. I'm good here. I'm going to resolve this thread.